acmw-db-app/pages/api/ghc/[pid].js

```diff
@@ -9,6 +9,30 @@ async function checkExternalScholarship(email) {
   return result.rows[0].external_sch;
 }
 
+// POST /api/ghc/enter-external-scholarship
+async function enterExternalScholarship(name_dot_num, id, type, description) {
+  let prefix;
+  if (type === "External Scholarship") {
+    prefix = "Ext=";
+  } else if (type === "Alternate Requirement") {
+    prefix = "Alt=";
+  } else {
+    throw("Invalid type.")
+  }
+
+  try {
+    await pqQuery(`
+      UPDATE ghcSET
```

Comment:
Please put a space between ghc and SET. I'm not 100% sure that it's required but ghcSET makes me very uncomfortable